cloud-operator/templates/cloud-operator-deployment.yaml

```diff
@@ -88,6 +88,8 @@ spec:
               value: "{{ .Values.env.streamSuccessPeriods.connect }}"
             - name: STREAM_SUCCESS_PERIOD_AUTH
               value: "{{ .Values.env.streamSuccessPeriods.auth }}"
+            - name: HTTPS_PROXY
+              value: "{{ .Values.httpsProxy }}"
```

Comment:
Add a test like https://github.com/illumio/cloud-operator/blob/df44b593931b44dd34b00bfa7b25eb248c9f90a5/cloud-operator/tests/cloud-operator-deployment_test.yaml#L81-L86
To check that this is properly set.